### ISSUE TITLE
HC Black: Links in launch pad and target page link in details view do not have sufficient contrast in both normal and on hover states

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#3476]
## Describe the bug
The inks in launch pad and target page link in details view do not have sufficient contrast against the background in both normal and hover states 

**To Reproduce**

1. Turn on windows high contrast and select High Contrast Black
2. Launch the AI Web Playground in Edge
3. Observe/Inspect the blue link do not have sufficient contrast

## Expected behavior

The links text needs to have a contrast of at least 4.5:1 in both normal and hover states

## Screenshots
![image](https://user-images.githubusercontent.com/22068434/96296286-e5895280-0fa3-11eb-862a-16d0b59e5116.png)

![image](https://user-images.githubusercontent.com/22068434/96296315-f1751480-0fa3-11eb-989c-c12c8734d65c.png)

## Context

- OS Name & Version: Windows 10 Enterprise, Version: 20H2, Build 19042.508
- AI-Web Version & Environment: Playground Version 2020.10.13.2130
- Browser Version: Microsoft Edge 86.0.622.43 (Official build) (64-bit)
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by ferBonnin** [comment]
@jlperkins can you help us verify if this issue still happens after fluent update?

**Comment by jlperkins** [comment]
These contrast issues still persist in our playground extension; however, in our latest dev version which includes the fluent update, this contrast issue appears to be resolved for Windows 11 HC modes. Please note that I am unable to test the Windows 10 HC modes since I have already upgraded, so someone still running Windows 10 will need to verify these changes as well.  Please see the images below showing the popup and target page text lines which now have enough contrast. 

However, with this change, for HC themes 1 and two (aquatic and desert), there is now less indication that those lines of text are links, as the link text and the body text are now nearly the same color. This is still an improvement as there is now sufficient contrast between the link text and the background, and hovering over the links still puts an underline underneath them, so they are still identifiable as links, just less obviously so...  Is this an issue that should be addressed? The other two Windows 11 HC themes, dusk and night sky, also have sufficient contrast, but also use a distinctly different color than the body text. 

![image](https://user-images.githubusercontent.com/33770444/165604765-bbfe23d5-5974-4d70-8c8d-15b7c7cd557f.png)
[Theme 1, aquatic, popup]

![image](https://user-images.githubusercontent.com/33770444/165604813-ab678a37-16c9-4df4-ac82-1f2cdbd22b9d.png)
[Theme 1, aquatic, target page line]

![image](https://user-images.githubusercontent.com/33770444/165605488-e408a219-d1a3-4a36-85a7-4c6dbfa9f786.png)
[Theme 2, desert, popup]

![image](https://user-images.githubusercontent.com/33770444/165605521-d4f0a2aa-e45c-40f0-a1d8-903807b5e5ea.png)
[Theme 2, desert, target page line]

![image](https://user-images.githubusercontent.com/33770444/165607807-9576b1da-3211-46fb-babc-3bda22a8520d.png)
[Theme 3, dusk, popup]

![image](https://user-images.githubusercontent.com/33770444/165607928-5c61800f-850c-4bb3-b8e1-70e26cd3a0dc.png)
[Theme 3, dusk, target page line]

![image](https://user-images.githubusercontent.com/33770444/165608076-1af3159c-e14b-4d92-b32c-af04024229bb.png)
[Theme 4, night sky, popup]

![image](https://user-images.githubusercontent.com/33770444/165608189-4ccf87c0-b6b5-4aa5-b975-1d4a02f97e31.png)
[Theme 4, night sky, target page line]

**Comment by DaveTryon** [comment]
Fix has been merged into main, will close once it gets deployed to Canary (deployment pipeline has been slow lately)

**Comment by DaveTryon** [comment]
This fix is in Canary 2023.4.3.1627. Closing.
